File: Data Structures & Algorithms/climbing-stairs/submission-0.cpp
class Solution {
public:
    int climbStairs(int n) {
        if(n==1 || n==2)return n;
        int a=0,b=1,c;
        for(int i=0;i<n;i++){
            c=a+b;
            a=b;
            b=c;
        }
        return c;
    }
};

File: Data Structures & Algorithms/find-the-difference-of-two-arrays/submission-0.py
class Solution:
    def findDifference(self, nums1: List[int], nums2: List[int]) -> List[List[int]]:
        l1=[]
        a=[]
        b=[]
        for i in nums1:
            if i not in nums2 and i not in a:
                a.append(i)
        l1.append(a)
        for i in nums2:
            if i not in nums1 and i not in b:
                b.append(i)
        l1.append(b)
        return l1

        